bot/exts/utilities/issues.py

```diff
@@ -139,15 +139,19 @@ async def fetch_issues(
             log.trace(f"PR provided, querying GH pulls API for additional information: {pulls_url}")
             async with self.bot.http_session.get(pulls_url) as p:
                 pull_data = await p.json()
-                if pull_data["draft"]:
-                    emoji = Emojis.pull_request_draft
-                elif pull_data["state"] == "open":
-                    emoji = Emojis.pull_request_open
-                # When 'merged_at' is not None, this means that the state of the PR is merged
-                elif pull_data["merged_at"] is not None:
-                    emoji = Emojis.pull_request_merged
-                else:
-                    emoji = Emojis.pull_request_closed
+                try:
+                    if pull_data.get("draft"):
+                        emoji = Emojis.pull_request_draft
+                    elif pull_data.get("state") == "open":
+                        emoji = Emojis.pull_request_open
+                    # When 'merged_at' is not None, this means that the state of the PR is merged
+                    elif pull_data.get("merged_at") is not None:
+                        emoji = Emojis.pull_request_merged
+                    else:
+                        emoji = Emojis.pull_request_closed
+                except KeyError:
+                    # couldn't get pull data
+                    emoji = Emojis.x_square
```

Comment:
Where is key error raised? All are using `.get()` which returns `None` if the key isn't present on the dict object (it doesn't raise a `KeyError` like `pull_data["draft"]` does).